### PR TITLE
Add use_openvino_from_location option

### PR DESCRIPTION
**Pull request by kanvi-nervana**
Rename use_prebuilt_openvino --> use_openvino_from_location to be consistent with TF build options at the bridge
Move build steps to a separate file build_ov.py

### CODE REVIEW

File: build_ngtf.py [hunk 1]
```diff
@@ -351,9 +351,13 @@ def main():
             "-DENABLE_SPEECH_DEMO=FALSE",
             "-DCMAKE_INSTALL_RPATH=\"$ORIGIN\"",
             "-DCMAKE_INSTALL_PREFIX=" + os.path.join(artifacts_location,
-                                                     "openvino")
+                                                     "openvino"),
+            "-DENABLE_PYTHON=ON",
+            "-DPYTHON_EXECUTABLE=" + os.path.join(build_dir_abs,
+                                                  "venv-tf-py3/bin/python3"),
```
Comment (bani-intelaipg) [inline build_ngtf.py:357]: venv dir is defined in line 191, we could use that

File: build_ngtf.py [hunk 2]
```diff
@@ -351,9 +351,13 @@ def main():
             "-DENABLE_SPEECH_DEMO=FALSE",
             "-DCMAKE_INSTALL_RPATH=\"$ORIGIN\"",
             "-DCMAKE_INSTALL_PREFIX=" + os.path.join(artifacts_location,
-                                                     "openvino")
+                                                     "openvino"),
+            "-DENABLE_PYTHON=ON",
+            "-DPYTHON_EXECUTABLE=" + os.path.join(build_dir_abs,
```
Comment (bani-intelaipg) [inline build_ngtf.py:356]: Instead of forcing Python3, can we get/read it from current venv?

**Review by bani-intelaipg — CHANGES_REQUESTED** [review]

File: build_ngtf.py (hunk 2 above)
Comment (kanvi-nervana) [reply build_ngtf.py:356]: Our scripts assume python3 https://github.com/tensorflow/ngraph-bridge/blob/master/tools/build_utils.py#L125 so we can probably stick to it.

**Review by bani-intelaipg — APPROVED** [review]

File: build_ngtf.py [hunk 3]
```diff
@@ -328,39 +329,31 @@ def main():
...
-        openvino_src_dir = "./openvino"
+    if arguments.use_openvino_from_location != "":
+        print("Using OpenVINO from " + arguments.use_openvino_from_location)
+    else:
+        print("Building OpenVINO from source")
+        print(
+            "NOTE: OpenVINO python module is not built when building from source."
+        )
+        python_module_enabled = False
+
```
Comment (adk9) [inline build_ngtf.py:340]: Remove building of Python module. We do not use the OpenVINO Python API.

File: test/ci/buildkite/setup.yml [hunk 4]
```diff
@@ -6,7 +6,7 @@ steps:
       rm -rf /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID
       virtualenv -p python3 /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/venv 
       source /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/venv/bin/activate 
-      pip install -U pip yapf==0.26.0 pytest psutil keras==2.3.1
+      pip install -U pip yapf==0.26.0 pytest psutil keras==2.3.1 cython
```
Comment (adk9) [inline test/ci/buildkite/setup.yml:9]: Not needed.

File: build_ngtf.py [hunk 5]
```diff
@@ -328,39 +329,31 @@ def main():
...
-            "-DENABLE_SPEECH_DEMO=FALSE",
-            "-DCMAKE_INSTALL_RPATH=\"$ORIGIN\"",
-            "-DCMAKE_INSTALL_PREFIX=" + os.path.join(artifacts_location,
-                                                     "openvino")
-        ]
-
+        debug_enabled = False
         if arguments.debug_build:
-            openvino_cmake_flags.extend(["-DCMAKE_BUILD_TYPE=Debug"])
+            debug_enabled = True
```
Comment (adk9) [inline build_ngtf.py:352]: `arguments.debug_build` can be used directly below.

**Review by adk9 — CHANGES_REQUESTED** [review]

**Review by adk9 — APPROVED** [review]

**Pull request merged**